modules/installation-and-upgrade/pages/postgresql-requirements.adoc

```diff
@@ -0,0 +1,17 @@
+[[installation-postgresql-requirements]]
+= {postgresql} Requirements
+
+{postgresql} is the only supported database.
+Using a remote {postgresql} database or remote file systems (such as NFS) with the {postgresql} database is not supported.
+In other words, {postgresql} should be on the fastest available storage for {productname}.
```

Comment:
```suggestion
In other words, {postgresql} should be on the fastest available storage for {productname}.
```

I don't quite understand what is meant by this sentence. Can it be somehow rephrased? I'm happy to brainstorm it together.

Reply:
Maybe, we should say "storage device".  There are dedicated storage systems.  If supported, SUMA or Uyuni can make use of them.  That's how I understand it.  If I'm wrong, experts will hopefully propose a better text.